backend/user/src/routes/user.ts

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+
+import { getAttemptedQuestions, addAttemptedQuestion } from '@/controllers/questions';
+
+
+const router = express.Router();
+
+router.get('/:userId/attempted-questions/', getAttemptedQuestions)
+router.post('/:userId/attempt-question', addAttemptedQuestion)
```

Comment:
/attempted-question/get, /attempted-question/add with JSON POST payloads for both as well as support for userId lists